app/models/team.rb

```diff
@@ -0,0 +1,54 @@
+# == Schema Information
+#
+# Table name: teams
+#
+#  id         :bigint           not null, primary key
+#  teamname   :string(255)
+#  avatar     :string(255)
+#  motto      :string(255)
+#  school     :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  token      :string(255)
+#
+
+require 'file_size_validator'
+
+class Team < ApplicationRecord
+  has_and_belongs_to_many :users
+  accepts_nested_attributes_for :users, allow_destroy: true
+
+  # validates_presence_of :teamname
```

Comment:
qq: why this got commented out? (same as other commented out lines in this file)

Reply:
Someone wanted to create two teams with the same teamname (with different member), so these uniqueness validation were removed.
`validates_presence_of :teamname` seems unnecessary since we've validated its length.

Reply:
sure.